platform/install/quick-start-guide.mdx

```diff
@@ -2,67 +2,109 @@
 
+The **vCluster CLI** is the preferred method for deploying the platform to a Kubernetes cluster. It streamlines the installation process and ensures compatibility with your Kubernetes environment. If you later decide to manage your platform instance via GitOps, you can export the applied Kubernetes manifests or Helm chart configuration and store them in a Git repository.
 
-Using vCluster CLI is the preferred way to deploy vCluster Platform to a Kubernetes cluster. If you decide later
-on you want to manage your vCluster Platform instance via GitOps, you may export the applied Kubernetes
-manifests or Helm chart configuration and store this data in a git repository.
+## Prerequisites
 
-## 1. Download and Install vCluster CLI
+Before you begin, ensure that you have the following:
+
+- Admin access to a Kubernetes cluster: Your current kube-context must have administrative privileges. See [Accessing Clusters with kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for more information.
+- Helm v3 installed: Helm is required for deploying the platform. Refer to the [Helm Installation Guide](https://helm.sh/docs/intro/install/) if you need to install it.
+- `kubectl` installed: Kubernetes command-line tool for interacting with the cluster. See [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for installation instructions.
+
+## Download and Install vCluster CLI
 
 Use one of the following commands to download the vCluster CLI binary from GitHub:
 
 <InstallCLI />
 
-## 2. Deploy the vCluster Platform
-
-:::caution Prerequisites
-Please make sure you meet the following requirements
-before running this command:
-1. Current kube-context has admin access to the cluster
-2. Helm v3 must be installed
-3. kubectl must be installed
+:::tip
+After installation, verify the successful installation of the vCluster CLI by running `vcluster version` in your terminal. This command should display the installed version of the vCluster CLI.
 :::
 
-Confirm that you are using the correct host cluster kube-context via `kubectl config current-context` and then run:
+## Deploy the platform
+
+Before deploying the platform, ensure you are using the correct host cluster kube-context. A host cluster kube-context is the configuration that determines which Kubernetes cluster and namespace `kubectl` commands will interact with. To confirm your current context, run:
+
+```bash title="Get the current Kubernetes context."
+kubectl config current-context
+```
 
-```bash
+Once you've confirmed the correct context, deploy the platform by running:
+
+```bash title="Deploy the platform."
 vcluster platform start
 ```
-This will deploy vCluster Platform into the host `vcluster-platform` namespace.
+
+This will deploy the platform onto the host-cluster in the `vcluster-platform` namespace.
 The UI will automatically open in your browser and log you in. You will be asked for your user details to create the admin user.
```

Comment:
This content of the automatic UI and admin user ceration is duplicated in two places. This is the first mention.

Reply:
It's probably due to partial duplicating some of the text. Fix incoming.